core/unix/signal_linux_aarch64.c

```diff
@@ -306,6 +306,8 @@ mcontext_to_sigcontext_simd(sig_full_cxt_t *sc_full, priv_mcontext_t *mc)
         esr->size = sizeof(struct esr_context);
 
         struct sve_context *sve = (void *)((byte *)esr + sizeof(struct esr_context));
+        const struct sve_context sve_zero = { 0 };
+        *sve = sve_zero;
```

Comment:
Add a comment with some of what's in the PR description, to explain to future readers why everything is zeroed? That there are new fields in reserved spots that cause problems if left uninitialized.

Reply:
Done.